lib/games_manager.rb

```diff
@@ -79,8 +79,7 @@ def worst_season(team_id)
   end
 
   def average_win_percentage(team_id)
-    wins = 0
-    all = 0
+    wins = all = 0
```

Comment:
I like where you heads at, I did a little digging around and it looks like this doesn't work quite how we think it does. Check out this link https://makandracards.com/makandra/37107-beware-ruby-s-var1-var2-value-multiple-assignment